lib/capybara/spec/session/node_spec.rb

```diff
@@ -102,6 +102,12 @@
       expect { @session.first('//textarea[@readonly]').set('changed') }.to raise_error(Capybara::ReadOnlyElementError)
     end
 
+    it 'should use global default options', requires: [:js] do
+      Capybara.default_set_options = { clear: :backspace }
+      expect_any_instance_of(Capybara::Selenium::Node).to receive(:set_text).with('gorilla', clear: :backspace)
```

Comment:
Find the element first, and then set the expectation on `element.base` rather than using any_instance_of.  Also you want to expect on :set, not :set_text since set_text is selenium specific.